<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gui</class>
 <widget class="QWidget" name="Gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>545</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <item>
    <layout class="QHBoxLayout" name="modeSwitchLayout">
     <item>
      <widget class="QPushButton" name="switchToAdminButton">
       <property name="text">
        <string>Admin Mode</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="switchToUserButton">
       <property name="text">
        <string>User Mode</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="modeStack">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="adminWidget">
      <layout class="QVBoxLayout" name="adminLayout">
       <item>
        <widget class="QTableWidget" name="table">
         <property name="columnCount">
          <number>4</number>
         </property>
         <property name="horizontalHeaderLabels" stdset="0">
          <stringlist>
           <string>Breed</string>
           <string>Name</string>
           <string>Age</string>
           <string>Photo</string>
          </stringlist>
         </property>
         <column/>
         <column/>
         <column/>
         <column/>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="formLayout">
         <item>
          <widget class="QLineEdit" name="breedInput">
           <property name="placeholderText">
            <string>Breed...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameInput">
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Name...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ageInput">
           <property name="placeholderText">
            <string>Age...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="photoInput">
           <property name="placeholderText">
            <string>Photo link...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="buttonLayout">
         <item>
          <widget class="QPushButton" name="addButton">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="updateButton">
           <property name="text">
            <string>Update</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="undoButton">
           <property name="text">
            <string>Undo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="redoButton">
           <property name="text">
            <string>Redo</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="userWidget">
      <layout class="QVBoxLayout" name="userLayout">
       <item>
        <widget class="QLabel" name="dogDetails">
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="dogImageLabel">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>250</height>
          </size>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="userButtonLayout">
         <item>
          <widget class="QPushButton" name="nextButton">
           <property name="text">
            <string>Next</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="adoptButton">
           <property name="text">
            <string>Adopt</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="filterLayout">
         <item>
          <widget class="QLineEdit" name="filterBreedInput">
           <property name="placeholderText">
            <string>Breed</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="filterAgeInput">
           <property name="placeholderText">
            <string>Max Age</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="filterButton">
           <property name="text">
            <string>Filter</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="viewAdoptionListButton">
         <property name="text">
          <string>View Adoption List</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
